README.md

```diff
@@ -1,90 +1,66 @@
-
-See [docs/contributing.rst](docs/contributing.rst).
+Simply install the agent, configure it to communicate with LightStep Satellites, run your app, and then any [frameworks](https://docs.lightstep.com/docs/python-auto-instrumentation#section-frameworks), [data stores](https://docs.lightstep.com/docs/python-auto-instrumentation#section-data-stores), and [libraries](https://docs.lightstep.com/docs/python-auto-instrumentation#section-libraries) included in your app will send data to LightStep as distributed traces.
 
-### Testing
+## Requirements
 
+- Python: 2.7, 3.4, 3.5, 3.6, 3.7
 
-#### Environment
+## Installing
 
-The test suite requires many backing services such as PostgreSQL, MySQL, Redis
-and more. We use ``docker`` and ``docker-compose`` to run the services in our CI
-and for development. To run the test matrix, please [install docker][docker] and
-[docker-compose][docker-compose] using the instructions provided by your platform. Then
-launch them through:
+```bash
+pip install ls-trace
+```
 
-    $ docker-compose up -d
-
-
-[docker]: https://www.docker.com/products/docker
-[docker-compose]: https://www.docker.com/products/docker-compose
-
-
-#### Running Tests in docker
-
-Once your docker-compose environment is running, you can run the test runner image:
-
-    $ docker-compose run --rm testrunner
-
-Now you are in a bash shell. You can now run tests as you would do in your local environment:
-
-    $ tox -e '{py35,py36}-redis{210}'
+## Getting Started
 
-We also provide a shell script to execute commands in the provided container.
+The following `app.py` makes a web request:
 
-For example to run the tests for `redis-py` 2.10 on Python 3.5 and 3.6:
+```python
+#!/usr/bin/env python3
+import requests
 
-    $ ./scripts/ddtest tox -e '{py35,py36}-redis{210}'
+def get_url(url):
+    response = requests.get(url)
+    print(response)
 
-If you want to run a list of tox environment (as CircleCI does) based on a
-pattern, you can use the following command:
+if __name__ == "__main__":
+    get_url("https://en.wikipedia.org/wiki/Duck")
+```
 
-    $ scripts/ddtest scripts/run-tox-scenario '^futures_contrib-'
+Now let's run the application using `ls-trace-run`
 
-### Continuous Integration
+```bash
+# export configuration options
+export DD_TRACE_AGENT_URL=https://collector.lightstep.com:443
+# replace <service_name> with your service's name
+# replace <access_token> with your LightStep API token
+export DD_TRACE_GLOBAL_TAGS="lightstep.service_name:<service_name>,lightstep.access_token:<access_token>"
 
-We use CircleCI 2.0 for our continuous integration.
+# run the application
+ls-trace-run ./app.py
+```
 
+A trace from the application should be available in your [LightStep dashboard](https://app.lightstep.com/)
 
-#### Configuration
+## Next Steps
 
-The CI tests are configured through [config.yml](.circleci/config.yml).
+Check out https://docs.lightstep.com/docs/python-auto-instrumentation for more information
 
+## Support
```

Comment:
Do you want to mention the relationship between the ls-trace version and dd-trace version?

Reply:
done!